scripts/testdata/satellite-config.yaml.lock

```diff
@@ -172,6 +172,9 @@ kademlia.operator.wallet: ""
 # a node's ratio of successful audits
 # overlay.node.audit-success-ratio: 0.4
 
+# require distinct IPs when choosing nodes for upload
+# overlay.node.distinct-ip: true
```

Comment:
Please dont forget to give the Satellite Operators / Release Team/Manager a heads up, once this gets merged!